src/components/Number.js

```diff
@@ -1,20 +1,49 @@
-import { Component } from 'react';
-import styled from 'styled-components';
+import { Component } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
 
-const Container = styled.div`
-  padding: 10px;
-  border: 1px solid black;
-  width: 60px;
-`;
+const Styles = {
+  Container: {
+    padding: "20px",
```

Comment:
```suggestion
    padding: 20,
```
it defaults to `px`, so you don't have to explicitly add it when using inline styling